airbyte-cdk/python/unit_tests/sources/declarative/interpolation/test_macros.py

```diff
@@ -71,3 +71,7 @@ def test_timestamp(test_name, input_value, expected_output):
     timestamp_function = macros["timestamp"]
     actual_output = timestamp_function(input_value)
     assert actual_output == expected_output
+
+
+def test_utc_datetime_to_local_timestamp_conversion():
```

Comment:
Wow, TIL! 
It would be good to have a comment here explaining what the behavior is in `datetime.utc` and how `pytz` resolves the issue. Otherwise, imagine ourselves in half a year from now trying to figure out why we have this test at all.

Reply:
docsting added

Reply:
nit: can we move the comment to the source code? I think we look at it more frequently than we look at tests

Reply:
moved